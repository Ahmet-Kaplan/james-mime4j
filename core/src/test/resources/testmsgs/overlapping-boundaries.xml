<message>
<header>
<field>
Received: by WOMS INBOUNDER;
	Tue, 21 Jun 2011 09:48:25 +0900 (KST)</field>
<field>
Date: Tue, 21 Jun 2011 09:48:25 +0900 (KST)</field>
<field>
From: =?UTF-8?B?6rmA7Jqp7ISx?= &lt;yskim@uwiseone.com&gt;</field>
<field>
Sender: =?UTF-8?B?6rmA7Jqp7ISx?= &lt;yskim@uwiseone.com&gt;</field>
<field>
To: =?UTF-8?B?6rmA7Jqp7ISx?= &lt;yskim@uwiseone.com&gt;</field>
<field>
Message-Id: &lt;643025386.114751308617305650.JavaMail.root@localhost.localdomain&gt;</field>
<field>
Subject: Test mail</field>
<field>
MIME-Version: 1.0</field>
<field>
Content-Type: multipart/mixed; 
	boundary="----=_Part_22594_1940300205.1308617305650"</field>
<field>
X-WOInboundFlag: true</field>
</header>
<multipart>
<body-part>
<header>
<field>
Content-Type: multipart/alternative; 
	boundary="----=_Part_22594_1940300205.13086173056501"</field>
</header>
<multipart>
<body-part>
<header>
<field>
Content-Type: text/plain; charset=UTF-8</field>
<field>
Content-Transfer-Encoding: base64</field>
<field>
Content-Disposition: inline</field>
</header>
<body>
dGVzdC4uLg==</body>
</body-part>
<body-part>
<header>
<field>
Content-Type: text/html; charset=UTF-8</field>
<field>
Content-Transfer-Encoding: base64</field>
<field>
Content-Disposition: inline</field>
</header>
<body>
dGVzdC4uLg==</body>
</body-part>
</multipart>
</body-part>
<body-part>
<header>
<field>
Content-Type: text/html; charset=UTF-8</field>
<field>
Content-Transfer-Encoding: base64</field>
<field>
Content-Disposition: inline</field>
</header>
<body>
aW1wb3J0YW50IHN0dWZm</body>
</body-part>
</multipart>
</message>
