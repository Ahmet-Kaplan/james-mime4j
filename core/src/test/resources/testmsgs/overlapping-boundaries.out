Received: by WOMS INBOUNDER;
	Tue, 21 Jun 2011 09:48:25 +0900 (KST)
Date: Tue, 21 Jun 2011 09:48:25 +0900 (KST)
From: =?UTF-8?B?6rmA7Jqp7ISx?= <yskim@uwiseone.com>
Sender: =?UTF-8?B?6rmA7Jqp7ISx?= <yskim@uwiseone.com>
To: =?UTF-8?B?6rmA7Jqp7ISx?= <yskim@uwiseone.com>
Message-Id: <643025386.114751308617305650.JavaMail.root@localhost.localdomain>
Subject: Test mail
MIME-Version: 1.0
Content-Type: multipart/mixed; 
	boundary="----=_Part_22594_1940300205.1308617305650"
X-WOInboundFlag: true

------=_Part_22594_1940300205.1308617305650
Content-Type: multipart/alternative; 
	boundary="----=_Part_22594_1940300205.13086173056501"

------=_Part_22594_1940300205.13086173056501
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: base64
Content-Disposition: inline

dGVzdC4uLg==

------=_Part_22594_1940300205.13086173056501
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: base64
Content-Disposition: inline

dGVzdC4uLg==

------=_Part_22594_1940300205.13086173056501--

------=_Part_22594_1940300205.1308617305650
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: base64
Content-Disposition: inline

aW1wb3J0YW50IHN0dWZm

------=_Part_22594_1940300205.1308617305650--
